.github/workflows/latex-docker.yml

```
@@ -0,0 +1,55 @@
+name: Test LaTeX (docker image)
+
+on:
+  push:
+    branches-ignore:
+      - 'self/**' # "**" also matches remaining "/"
+    paths:
+      - 'latex-examples'
+      - '.github/workflows/latex-docker.yml'
+  pull_request:
+    paths:
+      - 'latex-examples'
+      - '.github/workflows/latex-docker.yml'
+  workflow_dispatch:
+
+jobs:
+  test-suite:
+    runs-on: ubuntu-latest
+    container:
+      # https://gitlab.com/islandoftex/images/texlive
+      image: registry.gitlab.com/islandoftex/images/texlive:latest
```

Which one is downloaded faster?
https://gitlab.com/islandoftex/images/texlive
```suggestion
      image: texlive/texlive:latest
```